user_manual/groupware/sync_thunderbird.rst

```diff
@@ -8,29 +8,38 @@ Synchronizing with Thunderbird
 Recommended method
 ------------------
 
+Contacts
+~~~~~~~~
+
 Since Thunderbird 102, the CardDAV protocol is natively supported.
```

Comment:
In this case, can't we just put this one level up, and say directly under `Recommended method` that this is supported natively in current versions (or since version 102) of Thunderbird.
Then we won't need line 30 either.

Reply:
Good point!